iib/web/migrations/env.py

```diff
@@ -43,8 +43,7 @@ def run_migrations_offline():
 
 
 def run_migrations_online():
-    """
-    Run migrations in 'online' mode.
+    """Run migrations in 'online' mode.
```

Comment:
I feel this was removed by mistake :) We are trying to standardize the format of docstrings. Can you please fix this?

Reply:
> I feel this was removed by mistake :) We are trying to standardize the format of docstrings. Can you please fix this?
Yes black in my local machine changed this. Reverted this back to how it is.